index.d.ts

```diff
@@ -274,3 +275,24 @@ export interface PickerProps {
 }
 
 export class Picker extends React.Component<PickerProps> {}
+
+declare class Form extends React.Component<ScrollViewProps> {}
+
+interface FormFieldRenderProps {
+  focusableRef: React.Ref<any>;
+  onSubmitEditing: (e: any) => void;
```

Comment:
I think React Native should have export a proper type, maybe we can use it.

Reply:
Sure. Let me check check